### PR TITLE
Converted both common/ImageSelector.js #492  and  common/Switch.js #490 to functional component.

### PR DESCRIPTION
**Pull request by CharlesElloit**
Hi, I have converted both ImagesSelector and Switch to functional component.

### CODE REVIEW

File: src/components/common/Switch.js [hunk 1]
```diff
@@ -9,41 +9,34 @@ import "styles/Switch.scss";
...
-  }
-
-  componentDidUpdate(prevProps) {
-    if (this.props.on !== prevProps.on) {
-      this.setState({
-        on: this.props.on,
-      });
+
+const Switch = (props) => {
+  const [state, setState] = useState({
```
Comment (lumisphere902) [inline src/components/common/Switch.js:14]: This works, but does _on_ need to be nested inside a state object?

**Review by lumisphere902 — CHANGES_REQUESTED** [review]
Hi Charles, thanks for contributing. There's just a small question I had about the Switch component. Other than that, LGTM to merge!

File: src/components/common/Switch.js (hunk 1 above)
Comment (CharlesElloit) [reply src/components/common/Switch.js]: Well, not at all but i intend to follow state and setState naming convention from class to functional component and it's very clear for someone who hasn't use hook, considering that we are actually using the same state variable name throughout the component. Plus there is no any side effect of using that.
**But, if required, i will change it**

Comment (lumisphere902) [reply src/components/common/Switch.js]: This is unlikely to matter for this particular component, but in general I think separating state is better than a catch-all state

**Review by lumisphere902 — APPROVED** [review]
LGTM

**Pull request merged**